lib/improver/cli/nowcast_accumulate.py

```diff
@@ -43,35 +43,31 @@
 
 @cli.clizefy
 @cli.with_output
-def process(input_cube: cli.inputcube,
-            advection_cubes: inputadvection,
-            oe_cube: cli.inputcube,
+def process(cube: cli.inputcube,
+            advection_velocity: inputadvection,
+            orographic_enhancement: cli.inputcube,
             *,
-            attributes_dict: cli.inputjson = None,
+            new_metadata: cli.inputjson = None,
```

Comment:
Can we go with "new_attributes" for these arguments please?  There is no longer a case for (or code to support) generic unconstrained metadata updates.

Reply:
I originally went for `attributes_config`, but then noticed `new_metadata` in `combine`, so tried to be consistent. But actually the `combine` one is not metadata at all - it configures bounds expansion on coords, so I renamed that one to `bounds_config` and went back `attributes_config` for the other ones.

Reply:
Thanks